### DISCUSSION TITLE
GLIBC_2.29 not found error in logs

### DISCUSSION THREAD

**Discussion by 3rdStng** [teslamate-org/teslamate#2299] [General]
Just updated my 1.24.x docker on unRAID to 1.25.1 and upon starting up, the logs repeat with errors about GLIBC_2.29 not being found. I was able to pull back down the 1.24.2 image and my WebUI starts up just fine.

![image](https://user-images.githubusercontent.com/24888063/149045635-ab42182e-d3fb-4a08-a228-c8df7da2ef67.png)

**Comment by Early-Adapter** [comment]
I updated my docker on my Synology to 1.25.1 but I also can't see my WebUI.
When I checked the logs I found the same errors about GLIBC_2.29 not being found like this.

/opt/app/erts-12.2/bin/beam.smp: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /opt/app/erts-12.2/bin/beam.smp)

But I removed previous docker images using this command before I found this error, so I can't pull back my environment. 
   > docker image prune
   
So, I was able to modify my docker-compose.yml file to get the docker image 1.25.0, not the latest, and pull and up the docker image again.
I think this problem needs to be corrected quickly so that no one is suffering like me.

**Comment by 3rdStng** [comment]
Issue has been resolved.

https://github.com/adriankumpf/teslamate/issues/2300
